<launch>
    <node name="delta_odom" pkg="delta_odom" type="node" output="screen" >
        <rosparam command="load" file="$(find delta_odom)/params/basic.yaml" />
        <remap from="pose_stamped" to="/rail/nwu/pose_stamped"/>
    </node>
</launch>